### DISCUSSION TITLE
Format of GH_PAT value

### DISCUSSION THREAD

**Discussion by blairboy362** [BetaHuhn/repo-file-sync-action#234] [Q&A] [answered]
Hi folks.

I just want to clarify the format of the GH_PAT input value. Should it include the username? So it would be `username:PAT`?

I ask because I recently saw an issue where a workflow was throwing this error:

```
fatal: could not read Password for 'https://***@github.com': No such device or address
```

Which makes sense as git, when performing a clone, is attempting to prompt for a password. Including the username in the GH_PAT variable switched to a different error, an authentication failure, which I imagine is because the PAT is expired or has otherwise been revoked (something I need to look into anyway, but that's for another day). I wanted to raise this before I change our workflow configurations in case I'm missing something obvious somewhere.

Thanks.

**Comment by BetaHuhn** [comment, marked as answer]
The `GH_PAT` should be set to the value you get from GitHub after [generating a personal access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) token in their UI. No need to prepend it with your username.
